### DISCUSSION TITLE
User Guide

### DISCUSSION THREAD

**Discussion by Suitestallion** [burkybang/SuiteAdvanced-Feedback#175] [General]
Greetings,
Is there a User Guide, or the new things I can do now that I have SuiteAdvanced Plug in.

**Comment by Serendibb** [comment]
I keep this list, which is probably outdated, as one of my browser tab. This was written by the creator about 3 years ago? 
https://docs.google.com/document/d/e/2PACX-1vRhUQNIHNcuSi3GNEdxg7KnbqSkZCUqeh4yJgScLLD1r7eR-9zn7yTFDKX1peQ9X5AXK34cPcMwFjwY/pub

**Comment by burkybang** [comment]
Thanks @Serendibb for providing that link. Yes, it's a bit outdated and not very pretty. I haven't had time to document everything as it's so massive.

You can look through the release notes below for detailed changes since December 22, 2023.
https://suiteadvanced.com/releases

Other than that, there's a Changelog subtab in SuiteAdvanced settings that has everything documented since the beginning of time.
